test/new-e2e/utils/payloadtesthelper/fixtures/api_v2_series_dump.txt

```diff
@@ -0,0 +1 @@
kU4KqDkUmntisSOPDmVd/9/KZYemFlNAdC9BtIJuadQKMANezKeGaCXC1IQ2w/DyHpcxkaQFtAVJb1YlpWVBXP+NAp1+CATpPcehMMulUlylcJ/MwQRSSJNh8lk3mJiYZuAgxtZMwv8IR41amfRuSRaRBVWtAbSERHs+nJJNHA5msmQ4O5vJXODwnmDzboFeZ3JlBkhPpV09wCO6PEeUkMu4xDpf3irWBJGg8yaFxu2A3Sqw1fpFDphCRQGtsTKwSqEG0mAilx/vK6a7hkQSRpzlE3ic8xZIbHF8eZLHzB/5XB0ntxgA1igyE6wF2Jzjs3h1RcTbEiTSf5wapkOAi/K2UjOTIgD7KIfqeyMpMKqSITWJnmwSoe9sTBYRCPTveNjErTxXl3CTR5OL+OqJyMSdXpwt5AKsTmxRpfMS3sHXI+/daJU1LSB04dRjQ4XNsbvuiuF27m2SvTg+yab7LXrYpNGQMqYxqQIMyXfKHvNqEOLvd+6y8CGkjB2ersFiceFt5+Jj3N+PDmb978J/qG350Hb5W0mdqc9NzIfj909/i4zfNDtQRH9VPceW0kgz9UmG2MD0zlQuh/3E2/o4efIM62W0+LLJmKG7mWNTPf0mRNDb03h5o7BgqadG3BY24LeBjbCPjHUaoZsmaQ4ibntCmoNpEdgPqTOQhRVL7Vl7y2WsYMam7oQVx9aFr+lznW6BOhDocE+HR3ekDV8lkCU5VS7CQzwGYZ5mwnlWkat8yiQXZVer+uG6I6LPvWYWX/WLkdVNbgCSssDiFG865o14LM6VCSYdVnu79Fm4v4x7oev+JyW3JPen5UP+GBGrMtXCcc1Wpw4JmYkwJ0BZkuGRBjaoGWGLCnlekB9iwt4TCUs1yMVUCveHIxN6SRNuMCLGrCyc7o5SPHFsRjSfKCdHedoRt4h2hMeDFNDMvfTRuQQmkoFtNiew3pVwxN1MqhPSZ2DVEumvIrdU+c0V8BQlpaW0PjrLxs6GB9C/4seaFwZU8PJTibauv/kjC+mNvSONUUYTykXM80fVYy9XmSnotbP1u9K8V30CPWMlERWBqbSS7IoKe4PQt1XKluSmeHwv9CxZvTdyoaBG+RGyhtzMHMkUGNB0qMpgvEu5RLEovoZmZTKmEHFWvVqlEp1hsl2iYS1HrsGvpvrj5JRAShbcTiJ0Bu2arwNCOwBXZjiKuG0QeP0IuF1ylLiUwA5OpCpEUir140z4W7363TN6S5sYjdjOnG2jmyZR1VJpFjI0YFwd7GlUSh/SoncSNSjLJKyLUxpNX4GJbMnEXWeY2fqqJujKwkdBHzdi+Qh+qlcg03/hVlKgT4GObpdtaJm7W4SLSYLnLUjk/W5CLOTknF0e329iwZFS7x4LkGAXUsbNZ74aP6iJ0/2V2n4+oYmBA2wK92NnrEaaQtMZ1NrP2kwqaCNjl7JhrUG0RWyfS+twzOpWvgtLUXCmfr5ayu73PLWLWUBOW6xBDgrRJYjoUkoAMrwdleHlTHBk867K9bB+37kMGvAvbFGANblCLq++r8t+9R+ely7F83dYK+RHHqEKSeheHhoYyirQSK6tHWy7O6wbC11RafzuWDnvbtw799HaqBqIU6mr7ziVPkwn//0Xf7hVQ77qmCXRIyppJ4P9FqocMRTo52EY4sp4x3gFgUEkzpEgnYilxfUuJYxaE+gZFuGFdBHduDUKDy6urq3+dl85v5jzq4Hvcpibb0fFPsjp6MmX7ukoI63CpXEhUpqzkO1g2azVer/l6rLTxR6j/e7ZqmBhRk0Qk0Fu6x7gS/T7W5N3VIZHsYAKB1gDVZkrXjGHfWem3sXz0+yjVrAMOIXfYFFTT7d3jg50SKWiQrVT+fnob5Ho/bB98e1tkcSAt8tSA19IULRlvxmoTmZYUB1G9kcSiMTgh2o3kyQGnQkiRDC4ZHo0lRtLJkIrD2i2K9MRgZ0uqMuKNWO0hyTPIIGVoN4fnY2XC7sCx3XtfTdUdAigy8TW5T//imUuN9995pSL9e2BZMvgB8udYvu/sPEuk/CTRszadS++uGZwdPWtmPtk1c2trzXQsnlzxrmQlv1qk55CyOrD+kIKd4tKoNi99L72l050mo93FU8oNonri5UzSu2gato94N0+SViAd3ePUzxXpALt0trVuvjoWRQYMgLyxBtLIjM1LChtEwtSSzVrNhsEnXv/mIYx9YuW1kf1nKtJBNsJoi4a7e8nYJIYcfLbjHx5SYcu5PQ/ov4qJNB2C9Mq9P9m78spLFekuNtKNvZzCoMo//OXii1/7aUVEZQ8qzLObVlxGr4uDhDGpef/+WJE+xAa5IcAkDOWb1gel35y9XJEORT98oFmOd7Pe3ZWDjyX+s6MX7VCkD7OxtrewDDcLbXeZCwD9Zw3ouZM/VqSPsIHGOoAsUIcqdcFsBtrQqduEaNkPA0umJgVNBtC/Vk8rIlp5QJZG1RqBYS6+8Iezz5w+9GpFupONlenEYuB4vh36giKC40rF8+2R776oSHewcUYCset6VW0d5uZLP1NEKs2FqTq1GiRMqEblEUW6lw21uwPKboKqSqDWBMjg5FXjLycV6TAbclcHpGYaBqJqExmIMuAuHP+BkPZeDDmiDd1/tyLdzVt6OtJdoyqzdj7w4Or5l078qSJahBrQut86XGe15+kfCVez3QDsQF48ToRTyw0kA2bpk08pIuPAU5vr2dDFX1dECXFK7/LCQsxsolBXAatNHbpwQeE9N3xrC65my+5SoDYA1pvQlkGzaWreG9IO7kqy4cwVbzs6Fv3YYjcqwqpjBz2/0tHoSUWkNLtBe2qgE/IbisirN92Q3W9RdSLu475FtS9KTx7T73DeZt8T/FoBa5k2TCy3zh/ZjP/z2nmkmN0ZOKCoBvi0Scqz2bGujxE2SXmq59ctQLVGOfi83JvB44Ep/BUIxMEY4brQVyBwDHLegt6PVVNFKlZ1FapLcpdUYNnbVlrfyhLIZM/E/wIAAP//kW7fcQ=="]}
```

Comment:
Where does this format come from? How it is related to the fake intake?

Reply:
This is a dump from a curl to fake intake running on `ddev` on my machine - contains no sensitive infos, other than my laptop name, considered that safe enough to upload it. Can use manually created payloads instead of a fixture

Reply:
(adding a comment in code)